DNN Platform/DotNetNuke.Abstractions/Skins/SkinType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.UI.Skins;
+
+/// <summary>
+/// The type of a skin.
+/// </summary>
+public enum SkinType
+{
+    /// <summary>
+    /// The skin is a portal skin.
+    /// </summary>
+    Portal = 0,
+
+    /// <summary>
+    /// The skin is an admin skin.
+    /// </summary>
+    Admin = 1,
```

Comment:
Do we want to rename these to `Site` and `Edit` (matching `Dnn.PersonaBar.Themes.Components.ApplyThemeScope`)? It took me a few minutes to figure out what these meant, since we don't use these terms/concepts in the same way anymore.

Reply:
Should I rename the following methods in ISkinService as well?

| Current                     | New                       |
|-----------------------------|---------------------------|
| `GetDefaultAdminContainer`  | `GetDefaultEditContainer` |
| `GetDefaultAdminSkin`       | `GetDefaultEditSkin`      |
| `GetDefaultPortalContainer` | `GetDefaultSiteContainer` |
| `GetDefaultPortalSkin`      | `GetDefaultSiteSkin`      |